cfme/tests/cli/test_appliance_console.py

```diff
@@ -374,3 +374,57 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
                 logger.info("{}: no result".format(rule))
         else:
             logger.info("{}: rule not found".format(rule))
+
+
+def test_appliance_console_dhcp(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '1' configure DHCP, 'y' confirm IPv4 configuration, 'y' IPv6 configuration."""
+    command_set = ('ap', '', '1', '1', 'y', 'y')
+    unconfigured_appliance.appliance_console.run_commands(command_set)
+
+    def appliance_is_connective():
+        unconfigured_appliance.ssh_client.run_command("true")
+    wait_for(appliance_is_connective, handle_exception=True, delay=1, timeout=30)
+
+    soft_assert(unconfigured_appliance.ssh_client.run_command(
+        "ip a show dev eth0 | grep 'inet\s.*dynamic'"))
+    soft_assert(unconfigured_appliance.ssh_client.run_command
+        ("ip a show dev eth0 | grep 'inet6\s.*dynamic'"))
+
+
+def test_appliance_console_static_ipv4(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '2' configure static IPv4, '' confirm default IPv4 addr, '' confirm default netmask,
+    '' confirm default gateway, '' confirm default primary DNS, '' confirm default secondary DNS,
+    '' confirm default search order, 'y' apply static configuration.
+    """
```

Comment:
Same here for the doc string 
""" Command set - 
     'ap' launches appliance_console
     '' clears info screen
     '1' configure network
     '2' configure static IPv4
     '' confirm default IPv4 addr
     '' confirm default netmask
     '' confirm default gateway
     '' confirm default primary DNS
     '' confirm default secondary DNS
     '' confirm default search order 
     'y' apply static configuration
"""